Packs/PAN-OS/Integrations/Panorama/Panorama.py

```diff
@@ -6215,6 +6215,36 @@ def get_anti_spyware_best_practice_command():
     })
 
 
+def apply_dns_signature_policy_command(args) -> CommandResults:
+    anti_spy_ware_name = args.get('Anti_spyware_profile_name')
+    edl = args.get('DNS_signature_source')
+    action = args.get('Action')
+    packet_capture = args.get('Packet_capture', 'disable')
+    params = {
+        'action': 'set',
+        'type': 'config',
+        'xpath': f"/config/devices/entry[@name='localhost.localdomain']/device-group/entry[@name='{DEVICE_GROUP}']"
+                 f"/profiles/spyware/entry[@name='{anti_spy_ware_name}']",
+        'key': API_KEY,
+        'element': '<botnet-domains>'
+                   f'<lists>'
+                   f'<entry name="{edl}"><packet-capture>{packet_capture}</packet-capture>'
+                   f'<action><{action}/></action></entry>'
+                   f'</lists>'
+                   f'</botnet-domains>'
+    }
+    result = http_request(
+        URL,
+        'POST',
+        params=params,
+    )
+    res_status = result.get('response', {}).get('@status')
+    return CommandResults(outputs=result,
+                          outputs_prefix='Panorama.ApplyDNS',
+                          readable_output=f'**{res_status}**',
+                         )
```

Comment:
```suggestion
    return CommandResults(outputs=result,
                          outputs_prefix='PAN-OS.ApplyDNS',
                          readable_output=f'**{res_status}**',
                          raw_response=result
                         )
```

does the api returns just 'success'? is that why there is no table here? what is the string which is being returned for the ```res_status``` variable?
Also are we sure we want to return in the ```outputs``` the actual api result to the context? maybe we do not need it, lets discuss.